### PR TITLE
Updates JS implementation to latest Sass one and updates to use latest t...

### LINKED ISSUE #14: JS functionality has diverged from Sass
It looks like js needs updating to match https://github.com/Financial-Times/o-assets/pull/9 @AlbertoElias

### PR DESCRIPTION
**Pull request by AlbertoElias**
...esting tech

Closes #14

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: node_js
 node_js:
   - "0.10"
 before_install:
-  - npm install -g https://github.com/Financial-Times/origami-build-tools/tarball/master
-  - origami-build-tools install
+  - npm install -g Financial-Times/origami-build-tools#node-0.10
+  - obt install
```
Comment (matthew-andrews) [inline .travis.yml]: i've done this https://github.com/Financial-Times/next-build-tools/pull/16

**Comment by matthew-andrews** [comment]
[ignore my comment, thoguht this was next!]

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		browsers: ['PhantomJS'],
+
+
+		// Continuous Integration mode
+		// if true, Karma captures browsers, runs the tests and exits
+		singleRun: true,
+
+		browserify: {
+			debug: true,
+			transform: [ 'debowerify' ]
```
Comment (kaelig) [inline karma.conf.js]: No babelify?

Comment (AlbertoElias) [inline karma.conf.js]: I didn't really use es6 here as we still haven't decided on it

Comment (kaelig) [inline karma.conf.js]: Fair enough!

**Comment by kaelig** [comment]
I can't see any Sassy code in there, did I miss something?

**Comment by AlbertoElias** [comment]
@kaelig nope, it's all Javascript. The Sass code was up to date, but the Javascript wasn't up to date with the Sass

**Comment by kaelig** [comment]
Tested locally, works well! :+1:

**Pull request merged**